exporter/main.py

```diff
@@ -158,6 +161,10 @@ def handler(*args, **kwargs):
         os_ram_overcomit_ratio)
     collectors.append(hypervisor_stats)
 
+    if 'switches' in config:
```

Comment:
How will this deal with additional switches that are not corsa switches? Perhaps the configuration needs to account for multiple switch types. This is one of those things that you can also solve later, but if you already anticipate needing additional switch types (for example, to query the Juniper, or the Dell switches), it is good to be proactive.

Reply:
I'll make it more like the config file in metric collector